src/essentia/essentiamath.h

```diff
@@ -1200,6 +1200,142 @@ std::vector<std::vector<T> > pairwiseDistance(const std::vector<std::vector<T> >
   return pdist;
 }
 
+/**
+ * Sets `squeezeShape`, `summerizerShape`, `broadcastShape` to perform operations 
+ * on a Tensor with the shape of `tesor` along the `axis` dimension.
+ */
+template <typename T>
+void tensorGeometricalInfo(const Tensor<T>& tensor, int& axis,
```

Comment:
Do we need to group computation for `squeezeShape`, `summarizerShape`and `broadcastShape` together in one method?

I guess it is fine if we use this mostly as an auxiliary method.

Reply:
Yes, I thought it could be a useful auxiliary method in case we need to implement more axis-wise tensor operations in the future.